### PR TITLE
Upgrade to upcoming 4.0 release.

### PR DESCRIPTION
**Pull request by twiecki**
Preperation for the upcoming 4.0 release, do not merge yet.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by twiecki** [comment]
@conda-forge-admin, please rerender

**Comment by twiecki** [comment]
@conda-forge-admin, please rerender

**Comment by twiecki** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,67 +1,66 @@
...
+    - m2w64-toolchain  # [win]
 
 test:
   imports:
     - pymc
-    - pymc.database
-    - pymc.examples
+    - pymc.backends
+    - pymc.distributions
+    - pymc.glm
```
Comment (michaelosthege) [inline recipe/meta.yaml:41]:
```suggestion
```

**Review by michaelosthege — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,67 +1,67 @@
...
     - pip
   run:
     - python
-    - {{ pin_compatible('numpy') }}
-    - scipy
+    - aesara ==2.3.2
+    - aeppl ==0.0.18
+    - arviz >=0.11.4
+    - cachetools >=4.2.1
+    - cloudpicklel
```
Comment (michaelosthege) [inline recipe/meta.yaml:26]:
```suggestion
    - cloudpickle
```

**Comment by twiecki** [comment]
@conda-forge-admin, please rerender

**Comment by twiecki** [comment]
@conda-forge-admin, please rerender

**Comment by twiecki** [comment]
Alright, I think this looks good. CC @ocefpaf @fonnesbeck

**Review by fonnesbeck — APPROVED** [review]

**Pull request merged**